sklearn/manifold/tests/test_t_sne.py

```diff
@@ -1141,12 +1168,13 @@ def test_tsne_init_futurewarning(init):
 
 # TODO: Remove in 1.2
 @pytest.mark.parametrize("learning_rate", [None, 200.0])
-def test_tsne_learning_rate_futurewarning(learning_rate):
+@pytest.mark.parametrize("dtype", DTYPES)
+def test_tsne_learning_rate_futurewarning(learning_rate, dtype):
```

Comment:
Similar comment here: the future warning logic is completely unrelated to the dtype. Let's not over-parameterize this kind of tests.